CHANGELOG.md

```diff
@@ -1,5 +1,113 @@
 # Changelog
 
+## v0.17.0 (2017-8-2)
+
+### Enhancements
+
+#### :gear: Testing Electron applications ([testcafe-browser-provider-electron](https://github.com/DevExpress/testcafe-browser-provider-electron))
+
+We have prepared a browser provider that allows you to test Electron applications with TestCafe.
+
+Getting it to work is simple. First, install the plugin from npm.
+
+```sh
+npm install testcafe-browser-provider-electron --save-dev
+```
+
+Then create a `.testcafe-electron-rc` file that contains configurations for the Electron plugin.
+The only required setting here is `mainWindowUrl` - a URL or path to the main window page relative to the application directory.
+
+```json
+{
+    "mainWindowUrl": "./index.html"
+}
+```
+
+Place this file into the application root directory.
+
+Now you are ready to run tests. Specify the `electron` browser name and the application path
+when launching tests from the command line or API.
+
+```sh
+testcafe "electron:/home/user/electron-app" "path/to/test/file.js"
+```
+
+```js
+testCafe
+    .createRunner()
+    .src('path/to/test/file.js')
+    .browsers('electron:/home/user/electron-app')
+    .run();
+```
+
+To learn more about the Electron browser provider, see the plugin [readme](https://github.com/DevExpress/testcafe-browser-provider-electron).
+
+#### :gear: Concurrent test execution ([#1165](https://github.com/DevExpress/testcafe/issues/1165))
+
+To save your time spent on testing, we have added the capability to run tests concurrently in a single browser.
+
+By default, when you run tests in one or several browsers,
+TestCafe opens a single instance of each browser and runs all test sequentially within each instance.
+
+*Concurrency* allows TestCafe to open multiple instances of the same browser
+and distribute the test batch among these instances so that tests run in parallel.
```

Comment:
Enable concurrency and TestCafe will launch multiple instances of each browser. It will distribute the test batch among them. The tests will run in parallel.